newsfeed/queries.js

```diff
@@ -9,4 +9,17 @@ exports.createResturant  = function (name) {
                 }
         else console.log("Done");
     });
+}
+
+//14-I can add a restaurant to favourites.
+//The function takes as inputs the email of the user and the name of the restaurant 
+//and it gets the nodes of the restaurant and the user and creates a new relation called FAVORITES between the two nodes.
+exports.createrFavouriteUserRestaurant  = function (email,RestaurantName) {
+    db.query("MATCH (user:User {email: {ep}}), (rest:Restaurant {name: {rp}}) CREATE (user)-[:FAVORITES]->(rest);",
+     params = {ep:email,rp:RestaurantName}, function (err, results) {
+        if (err){  console.log('Error');
+                 throw err;
+                }
+        else console.log("Done");
+    });
 }
```

Comment:
@khaled-elsheikh Add the following condition, if the relation already exists then a new relation should not be added.